test/gtest/ucs/test_string.cc

```diff
@@ -9,6 +9,7 @@ extern "C" {
 #include <ucs/debug/memtrack_int.h>
 #include <ucs/datastruct/string_buffer.h>
 #include <ucs/datastruct/string_set.h>
+#include <ucs/sys/compiler.h>
```

Comment:
why needed? we didn't add new code to this file

Reply:
Right, but now we use `ucs_alloca` in array.h, and BlueField build fails to find that function unless explicitly include this header

Reply:
well, that's why i used UCS_STATIC_ASSERT rather than ucs_alloca

Reply:
Yes, exactly, but the consequence of using `UCS_STATIC_ASSERT` is that we cannot allocate array of non-compile-time constant on stack

Reply:
ok, let's keep it as is then